js/scripts/build-rpc-markdown.js

```diff
@@ -48,24 +43,26 @@ function error (log) {
 }
 
 function printType (type) {
-  return type2print.get(type) || type.name;
+  return type.print || `\`${type.name}\``;
```

Comment:
I think we need to decide on wether we want template string even if they require escaping `` ` ``.